### PR TITLE
Add new, versatile toolbar icon.

### LINKED ISSUE #1: Add nativeImage icons
Fix the icon so it is visible on dark toolbars on macOS. See http://electron.atom.io/docs/api/native-image/

### PR DESCRIPTION
**Pull request by zastrow**
Fixes #1

### CODE REVIEW

**Comment by zastrow** [comment]
## For visual reference

### Standard icon

icontemplate 2x

### Status change icon

show 2x

File: index.js [hunk 1]
```diff
@@ -9,15 +9,9 @@ const network = require('on-change-network');
...
-/*
-  Icon is Broadcast by Montu Yadav
-  https://thenounproject.com/term/broadcast/201837/
-  https://creativecommons.org/licenses/by/3.0/legalcode
-*/
-
 const params = {
   index: `file://${__dirname}/index.html`,
-  icon: './assets/icon.png',
+  icon: './assets/iconTemplate.png',
```
Comment (zastrow) [inline index.js]: This name change from icon to iconTemplate is the secret sauce to making icons work with the macOS dark toolbar. http://electron.atom.io/docs/api/native-image/#template-image

Comment (zastrow) [inline index.js:14]: This name change from icon to iconTemplate is the secret sauce to making icons work with the macOS dark toolbar. http://electron.atom.io/docs/api/native-image/#template-image

Comment (asimpson) [reply index.js:14]: I ❤️ you

**Comment by asimpson** [comment]
👍 

getinsertpic.com

**Comment by catheraaine** [comment]
pie

**Pull request merged**